src/app/core/services/auth.service.ts

```diff
@@ -30,13 +31,16 @@ export class AuthService {
 
   logIn(login: string, password: string) {
     if (login && password) {
+      this.loader.showLoader();
```

Comment:
do you think it is good way to copy/paste same showLoader/hideLoader in each service?) what if you will forget it? any ideas how to improve it? ❕ hint - interceptor